packages/utils/src/fhir-converter/convert.ts

```diff
@@ -60,11 +61,13 @@ async function convert(
     throw new Error(`File has nonXMLBody`);
   }
 
+  const noB64FileContents = removeBase64PdfEntries(fileContents);
+
   const unusedSegments = false;
   const invalidAccess = false;
   const params = { patientId, fileName, unusedSegments, invalidAccess };
   const url = `/api/convert/cda/ccd.hbs`;
-  const payload = (fileContents ?? "").trim();
+  const payload = (noB64FileContents ?? "").trim();
```

Comment:
why is this necessary here if we're doing it in `packages/lambdas/src/sqs-to-converter.ts`?

Reply:
It's used in the e2e tests

Reply:
nit: e2e tests of the converter

Also, shouldn't we using the same cleaning code the lambda uses, `cleanUpPayload`?